src/presenter/trip-event-presenter.js

```diff
@@ -0,0 +1,27 @@
+import { render } from '../render.js';
+import ListEvent from '../view/event-list-view.js';
+import FormItemEvent from '../view/event-edit.js';
+import { ITEMS_EVENT } from '../view/data-event-list.js';
+import ItemsEvent from '../view/event-items.js';
+// function createItemEvent(el){
+//   return el;
+// }
+
+export default class EventPresenter {
+
+  eventComponent = new ListEvent();
+
+  constructor({listContainer}){
+    this.listContainer = listContainer;
+  }
+
+  init(){
+    render(this.eventComponent,this.listContainer);
+    render(new FormItemEvent(), this.eventComponent.getElement());
+    render(new ListEvent(), this.eventComponent.getElement());
+
+    for(let i = 0; i < ITEMS_EVENT.length; i++){
```

Comment:
Просто 3 раза отрисовываем одинаковую вьюшку одного ивента. Когда у нас будут данные на руках мы для каждого объекта данных наших точек будем отрисовывать соответввующую вьюшку и передавать внутри данные и выводить их в шаблоне, пока просто так:
```suggestion
  for(let i = 0; i < 3; i++){
      render(new EventsItemView(),this.eventComponent.getElement());
    }
```